scripts/commit-msg

```diff
@@ -0,0 +1,26 @@
+
+#!/usr/bin/env bash
```

Comment:
nope. wasnt it you who said this has to be JS? this is a new project, lets do it right from the beginning please :+1: